### PR TITLE
nyxt: add page

### LINKED ISSUE #7408: page request: nyxt
- include a link to a web page about the command
https://nyxt.atlas.engineer/
- if available, add an existing source of examples
https://github.com/atlas-engineer/nyxt

```
# nyxt

> keyboard-driven web browser
> Nyxt is a keyboard-driven web browser for power users.
> Inspired by Emacs and Vim it is extensible in Lisp.
> More information: <https://nyxt.atlas.engineer>.

- Print system information:
`nyxt --system-information`

- List all profiles:
`nyxt --list-data-profiles`

- Set init.lisp file path:
`nyxt --init {{path/to/file}}`

- Change the auto-config.lisp;
`nyxt --auto-config {{path/to/file}}`
```

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

closes #7408 

Also I have no idea about this command, so feel free to commit any changes

### CODE REVIEW

File: pages/common/nyxt.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# nyxt
+
+> Nyxt is a keyboard-driven web browser for power users.
+> More information: <https://nyxt.atlas.engineer>.
+
+- Print system information:
```
Comment (marchersimon) [inline pages/common/nyxt.md:6]: This doesn't really feel like it's the most important flag. Maybe we should move it to the bottom?

File: pages/common/nyxt.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`nyxt --system-information`
+
+- List all profiles:
+
+`nyxt --list-data-profiles`
+
+- Set init.lisp file path:
```
Comment (marchersimon) [inline pages/common/nyxt.md:14]:
```suggestion
- Set the `init.lisp` file path:
```

File: pages/common/nyxt.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`nyxt --init {{path/to/file}}`
+
+- Change the auto-config.lisp:
```
Comment (marchersimon) [inline pages/common/nyxt.md:18]:
```suggestion
- Change the path to the auto-config file:
```

**Review by marchersimon — COMMENTED** [review]
I feel like this page is missing the more basic flags. Something like
```
- Open the `nyxt` GUI:

`nyxt`

- Start `nyxt` without loading the init file:

`nyxt --no-init`
```

**Review by marchersimon — APPROVED** [review]
As I already mentioned, this page really could use more basic examples, but otherwise it looks good

**Comment by CleanMachine1** [comment]
Never used the command at all, and to be honest, dont know anything about the command

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**